SaintCoinach/ex.json

```
@@ -11609,6 +11698,14 @@
         {
           "index": 9,
           "name": "HuntingLogExpReward"
+        },
+        {
+          "index": 11,
+          "name": "ItemLevelSync"
+        },
+        {
+          "index": 14,
+          "name": "cLevel"
```

```suggestion
          "name": "ClassJobLevel"
```

Is this "ClassJob" ?

---

supposedly crafterlevel?, apparently a level for crafting that is used to determine potency penalties/bonuses

---

Maybe CraftingLevel? C is too ambiguous.